config/mappings/assessment_mapping_schema.py

```diff
@@ -78,6 +78,7 @@
                     # covers:
                     # new add-another component with children support
                     # see https://github.com/communitiesuk/digital-form-builder/pull/161
+                    "integer",
```

Comment:
Maybe we could leave a comment
```suggestion
                    "integer",
                    # covers:
                    # numberField (when we have a number input we want as a whole number)
```

Reply:
Done